src/main/java/cl/transbank/webpay/webpayplus/WebpayPlus.java

```diff
@@ -101,52 +101,49 @@ public static WebpayPlusTransactionCreateResponse create(
             }
         }
 
-        public static WebpayPlusTransactionCommitResponse commit(String token) throws IOException, TransactionCommitException {
+        public static WebpayPlusTransactionCommitResponse commit(String token) throws TransactionCommitException {
             return WebpayPlus.Transaction.commit(token, null);
         }
 
         public static WebpayPlusTransactionCommitResponse commit(String token, Options options)
-                throws IOException, TransactionCommitException {
-            options = WebpayPlus.Transaction.buildOptions(options);
-            final URL endpoint = new URL(String.format("%s/%s", WebpayPlus.getCurrentIntegrationTypeUrl(options.getIntegrationType()), token));
-
+                throws TransactionCommitException {
             try {
+                options = WebpayPlus.Transaction.buildOptions(options);
+                final URL endpoint = new URL(String.format("%s/%s", WebpayPlus.getCurrentIntegrationTypeUrl(options.getIntegrationType()), token));
                 return WebpayApiResource.execute(endpoint, HttpUtil.RequestMethod.PUT, options, WebpayPlusTransactionCommitResponse.class);
-            } catch (TransbankException e) {
+            } catch (Exception e) {
```

Comment:
Isn't this change going in the opposite direction of what @goncafa did on #93 ?

Reply:
hey Leo, this is an unwanted change, anyways already fixed it, cheers!